src/math/Spherical.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @author bhouston / http://clara.io
+ * @author WestLangley / http://github.com/WestLangley
+ *
+ * Ref: https://en.wikipedia.org/wiki/Spherical_coordinate_system
+ *
+ */
+
+THREE.Spherical = function ( radius, phi, theta ) {
+
+	this.radius = ( radius !== undefined ) ? radius : 1.0;
+	this.phi = ( phi !== undefined ) ? phi : 0; // up / down towards top and bottom pole
+	this.theta = ( theta !== undefined ) ? theta : 0; // around the equator of the sphere
+
+	return this;
+
+};
+
+THREE.Spherical.prototype = {
+
+	constructor: THREE.Spherical,
+
+	set: function ( radius, phi, theta ) {
+
+		this.radius = radius;
+    this.phi = phi;
+    this.theta = theta;
+
+  },
+
+	clone: function () {
+
+		return new this.constructor().copy( this );
+
+	},
+
+	copy: function ( other ) {
+
+		this.radius.copy( other.radius );
+		this.phi.copy( other.phi );
+		this.theta.copy( other.theta );
+
+		return this;
+
+	},
+
+  add: function ( other ) {
+
+		this.radius += other.radius;
+    this.phi += other.phi;
+    this.theta += other.theta;
+
+    return this;
+
+  },
+
+  multiplyScalar: function( scalar ) {
+
```

Comment:
I am not a fan of this `multiplyScalar()` method. I would remove it, and inline the code where it is used.

I feel the same about the `add()` method.